.changes/unreleased/tinymce-TINY-10658-2024-02-22.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Improved
+body: Make table ghost element better reflect height changes when resizing
```

```suggestion
body: Make table ghost element better reflect height changes when resizing.
```

Do we care about dots anymore or was that a Brian thing?